CHANGES.md

```diff
@@ -34,6 +34,10 @@ To be released.
 
 ### Bug fixes
 
+ -  (Libplanet.Net) Fixed a bug where `Swarm<T>.PreloadAsync()`
+    had not thrown `OperationCanceledException` even cancellation
+    was requested.  [[#1796]]
```

Comment:
It would be good to link issue <https://github.com/planetarium/libplanet/issues/1547> as well.